arbnode/espresso_caff_node.go

```diff
@@ -238,6 +263,63 @@ func NewEspressoCaffNode(
 		fatalErrChan,
 	)
 
+	// Create a new EspressoKeyManager
+	// Get the EspressoTEEVerifier address from SequencerInbox contract
+
+	espressoTEEVerifierAddress, err := sequencerInbox.con.EspressoTEEVerifier(&bind.CallOpts{})
```

Comment:
there is one issue with this approach which you also mentioned on the slack, the current SequencerInbox doesnt have the latest version of EspressoTEEVerifier with your changes. How should we deal with this? Should we just accept the EspressoTEEVerifier as an argument

Reply:
So our plan will be to run 2 independent EspressoTEEVerifier contracts on the networks to support this caff node temporarily?